source/game/system/KPadController.cc

```diff
@@ -174,12 +166,14 @@ KPadGhostTrickButtonsStream::KPadGhostTrickButtonsStream() = default;
 
 KPadGhostTrickButtonsStream::~KPadGhostTrickButtonsStream() = default;
 
-bool KPadGhostTrickButtonsStream::readIsNewSequence(const u16 *sequence) const {
-    return (*sequence & 0xFFF) <= readSequenceFrames;
+bool KPadGhostTrickButtonsStream::readIsNewSequence() const {
+    u16 duration = currentSequence & 0xFF;
+    duration += 256 * ((currentSequence >> 8) & 0xF);
+    return duration <= readSequenceFrames;
 }
 
-u16 KPadGhostTrickButtonsStream::readVal(const u16 *sequence) const {
-    return *sequence >> 0xC;
+u8 KPadGhostTrickButtonsStream::readVal() const {
+    return (currentSequence >> 8) & 0x7F;
 }
```

Comment:
```suggestion
u8 KPadGhostTrickButtonsStream::readVal() const {
    return currentSequence >> 0xc;
}
```

Reply:
This causes a desync if I apply this change

Reply:
OK we cleared this up, Aiden misread